src/lib/Libattr/master_svr_attr_def.xml

```diff
@@ -1630,12 +1628,12 @@
 	<member_at_set>set_b</member_at_set>
 	<member_at_comp>comp_b</member_at_comp>
 	<member_at_free>free_null</member_at_free>
-	<member_at_action>action_power_provisioning</member_at_action>
-	<member_at_flags><both>MGR_ONLY_SET</both></member_at_flags>
+	<member_at_action>NULL_FUNC</member_at_action>
+	<member_at_flags><both>ATR_DFLAG_SvWR|ATR_DFLAG_MGRD</both></member_at_flags>
 	<member_at_type><both>ATR_TYPE_BOOL</both></member_at_type>
 	<member_at_parent>PARENT_TYPE_SERVER</member_at_parent>
 	<member_verify_function>
-	<ECL>verify_datatype_bool</ECL>
+	<ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
```

Comment:
Why the validation function verify_datatype_bool is removed.

Reply:
Power provisioning attribute is made read only now and gets set when you enable/disable power hook. So we dont need ecl validation any more.